book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rents
+
```

Comment:
Kindly, add a method for adding rentals for a book. Where the method receives `person`, `date`, and the book itself. That will be something like this:

```
def add_rental(person, date)
    Rental.new(date, self, person)
end
```

Make sure`:rents` will also be `attr_accessor`